app/containers/App/actions.js

```diff
@@ -1,11 +1,16 @@
 import {
+  LOAD_TOKEN,
   SHOW_LOADER,
   HIDE_LOADER,
   GET_USER_SUCCESS,
   GET_USER_FAILURE,
   SIGN_OUT,
 } from './constants';
 
+export const loadToken = () => ({
+  type: LOAD_TOKEN,
```

Comment:
Here you could read the local storage and then dispatch another action with the value of the token.